### ISSUE TITLE
`cSpell.enabledFileTypes` is still confusing me

### ISSUE THREAD

**Issue by cbrnr** [streetsidesoftware/vscode-spell-checker#4159]
This is a follow-up to #3836. The languages R and Julia have since been removed from the defaults, so setting

```
    "cSpell.enabledFileTypes": {
        "*": false,
    },
```

does not check these types now (which is good!). However, I can't figure out why Python files are still checked despite of this setting. What am I missing?

**Comment by Nxt3** [comment]
This setting was working for me and just started acting up today, using version 4.0.39.

```
  "cSpell.enabledFileTypes": {
    "*": false,
    "json": true,
    "jsonc": true,
    "markdown": true
  }
```

This should disable for all files except json, jsonc, and markdown. Yet it's making suggestions on `*.tsx` files now.

**Comment by Jason3S** [comment]
@cbrnr,

I am sorry that this issue has come back up again. It looks like some legacy code slipped into one of the dependencies. As a result, it populated `enabledLanguageIds` with a bunch of file types:

```js
        enabledLanguageIds: [
            'ada',
            'csharp',
            'go',
            'javascript',
            'javascriptreact',
            'json',
            'markdown',
            'mdx',
            'php',
            'plaintext',
            'python',
            'text',
            'typescript',
            'typescriptreact',
            'haskell',
            'html',
            'css',
            'less',
            'scss',
            'latex',
            'ruby',
            'rust',
            'shellscript',
            'toml',
        ],
```

These have been removed.
